bigbluebutton-web/grails-app/conf/spring/resources.xml

```diff
@@ -171,7 +171,7 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
         <property name="muteOnStart" value="${muteOnStart}"/>
         <property name="allowModsToUnmuteUsers" value="${allowModsToUnmuteUsers}"/>
         <property name="allowModsToEjectCameras" value="${allowModsToEjectCameras}"/>
-        <property name="breakoutRoomsEnabled" value="${breakoutRoomsEnabled}"/>
+        <property name="breakoutRoomsEnabled" value="${breakoutRoomsEnabled:true}"/>

```

Comment:
Why did you need to pass `true` for `breakoutRoomsEnabled`? Is that instead of an entry in bigbluebutton.properties?

Reply:
It is there to avoid warnings..

Since #14599, `breakoutRoomsEnabled` was removed from config by default.
So this set `true` as alternative value when `breakoutRoomsEnabled` is absent from config file.